src/components/container/sticky-bar/sticky-bar.scss

```diff
@@ -6,33 +6,32 @@
   --sbb-sticky-bar-padding-block: var(--sbb-spacing-responsive-xxxs);
   --sbb-sticky-bar-animation-duration: var(--sbb-animation-duration-5x);
   --sbb-sticky-bar-animation-easing: var(--sbb-animation-easing);
-  --sbb-sticky-bar-background-color: var(
+
+  display: contents;
+}
+
+:host([data-sticking]) {
```

Comment:
only defining --sbb-sticky-bar-sticky-background-color when it's really sticking, which makes the not selectors obsolete

Reply:
Love the new approach 😄